apps/src/fish/FishView.jsx

```diff
@@ -53,23 +47,98 @@ class FishView extends React.Component {
     mobilePortraitWidth: PropTypes.number.isRequired
   };
 
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      windowWidth: $(window).width(),
+      windowHeight: $(window).height(),
+      appWidth: $('#codeApp').width(),
+      appHeight: $('#codeApp').height()
+    };
```

Comment:
In general, we're trying to move away from jquery. Would you be able to use window.innerWidth and window.innerHeight here? Same for #codeApp (let me know if you'd like suggestions for an alternate way to find #codeApp width and height)